include/ClientSession.class.php

```diff
@@ -200,45 +195,50 @@ public function getNotifications()
     }
 
     /**
-     * Get the public address of a player
+     * Set the key to join server with client ip and port
      *
-     * @param int $peer_id id of the peer
+     * @param int $server_id id of the server
+     * @param int $address ip of client
+     * @param int $port port of client
+     * @param char $key aes 128 bit key in base64 of client
+     * @param char $iv initialization vector of the aes key
      *
      * @return array the ip and port of the player
      * @throws ClientSessionException if the request fails
      */
-    public function getPeerAddress($peer_id)
+    public function setJoinServerKey($server_id, $address, $port, $key, $iv)
```

Comment:
`$key` should be `$aes_key` and and `$iv` should be `$aes_iv`. Also update comments